_sass/_videos.scss

```diff
@@ -0,0 +1,24 @@
+.youtube {
+  background-color: #000;
+  position: relative;
+  cursor: pointer;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+
+  &::before {
+    content: "";
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='100%25' version='1.1' viewBox='0 0 68 48' width='100%25'%3E%3Cpath d='M66.52,7.74c-0.78-2.93-2.49-5.41-5.42-6.19C55.79,.13,34,0,34,0S12.21,.13,6.9,1.55 C3.97,2.33,2.27,4.81,1.48,7.74C0.06,13.05,0,24,0,24s0.06,10.95,1.48,16.26c0.78,2.93,2.49,5.41,5.42,6.19 C12.21,47.87,34,48,34,48s21.79-0.13,27.1-1.55c2.93-0.78,4.64-3.26,5.42-6.19C67.94,34.95,68,24,68,24S67.94,13.05,66.52,7.74z' fill='%23212121' fill-opacity='0.8'%3E%3C/path%3E%3Cpath d='M 45,24 27,14 27,34' fill='%23fff'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
@XhmikosR, I don't think this svg is minified (`version='1.1'` is redundant for example).

Reply:
I don't think SVGO removes it. But anyway, I'm having an issue with this approach. On mobile the user needs to click twice for the video to start playing, one for it to show and one to start playing.

So I made #98 as an alternative.

Reply:
See https://stackoverflow.com/a/15093243/796035